include/crab/analysis/inter/top_down_inter_analyzer.hpp

```diff
@@ -1032,6 +1032,104 @@ class top_down_inter_transformer final
     // crab::CrabStats::count("Interprocedural.num_calling_contexts");
   }
 
+  /// @brief group regions variables in parameter lists based on target's dsa
```

Comment:
What do source and target mean? Is it source = caller and target = callee ? If yes, use plese caller/callee